src/content/contribute/writing-a-loader.md

```diff
@@ -48,93 +39,111 @@ resolveLoader: {
 }
 ```
 
+Last but not least, if you've already created a separate repository and package for your loader, you could [`npm link`](https://docs.npmjs.com/cli/link) it to the project in which you'd like to test it out.
 
-## Examples
 
-``` javascript
-// Identity loader
-module.exports = function(source) {
-  return source;
-};
-```
+## Simple Usage
```

Comment:
```
## Usage

### Basic
...
### Advanced
...
```

Reply:
I'd actually prefer not to add unnecessary headers which also would require extra lead in content (i.e. not really a fan of header on top of header with nothing in between).

Reply:
kk